tests/FSharp.Data.Tests/Http.fs

```diff
@@ -88,16 +89,25 @@ let ``Setting timeout in customizeHttpRequest overrides timeout argument`` () =
 
     response.StatusCode |> should equal 401
 
-[<Test>]
-let ``4k+ bodies work`` () =
-    let bodyString2 = seq {for i in 0..4000 -> "x\n"} |> String.concat ""
-    let body2 = FSharp.Data.FormValues([("input", bodyString2)])
+let testFormDataSizesInBytes = [
+    4000    // previous test size
+    20000   // previous test size
+    40000   // > 80k, reported by user @danyx23 on full-framework
```

Comment:
note that we write two-bytes for each n here, so that's why the numbers are off from the reported numbers. newlines and such.